srml/contract/src/tests.rs

```diff
@@ -720,6 +720,22 @@ fn deduct_blocks() {
 			assert_eq!(bob_contract.rent_allowance, 1_000 - rent);
 			assert_eq!(bob_contract.deduct_block, 5);
 			assert_eq!(Balances::free_balance(BOB), 30_000 - rent);
+
+			// Advance 7 blocks more
+			System::initialize(&12, &[0u8; 32].into(), &[0u8; 32].into());
+
+			// Trigger rent through call
+			assert_ok!(Contract::call(Origin::signed(ALICE), BOB, 0, 100_000, call::null()));
```

Comment:
can we also test a second call at this block (I won't asker recursive contract call in transaction as I am really not sure it is doable with the test backend).

Reply:
the same call twice ?

Reply:
yes just to ensure rent is not payed twice when two call are done on same accountdb

Reply:
yes ok done